e2e/testdata/fn-eval/function-chain/.expected/exec.sh

```diff
@@ -0,0 +1,6 @@
+#! /bin/bash
+
+kpt fn source |\
+kpt fn eval - --image gcr.io/kpt-fn/set-namespace:v0.1 -- namespace=staging |\
+kpt fn eval - --image gcr.io/kpt-fn/set-label:v0.1 -- foo=bar |\
+kpt fn sink .
```

Comment:
does this allow testing sinking to a different directory?

Reply:
It doesn't support it directly, but we can sink to a different directory and copy back the resource to get diff.